configurevalidator/configurevalidator.go

```diff
@@ -0,0 +1,152 @@
+package configurevalidator
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+
+)
+
+var profile string
+var NumberRegion int
+var regions map[string]string
+var numbers[14]string
+var file os.File
+var usr *user.User
+var path string
+var exist bool
+
+func GetExist() bool{
+	return exist
+}
+
+func MakeArrayRegions () [14]string {
+	numbers[0]="us-east-2"
+	numbers[1]="us-east-1"
+	numbers[2]="us-west-1"
+	numbers[3]="us-west-2"
+	numbers[4]="ap-south-1"
+	numbers[5]="ap-northeast-2"
+	numbers[6]="ap-southeast-1"
+	numbers[7]="ap-southeast-2"
+	numbers[8]="ap-northeast-1"
+	numbers[9]="ca-central-1"
+	numbers[10]="eu-central-1"
+	numbers[11]="eu-west-1"
+	numbers[12]="eu-west-2"
+	numbers[13]="sa-east-1"
+
+	return numbers
+
+}
+
+func ShowRegions( ) {
+numbers=MakeArrayRegions()
+	fmt.Println("New configuration file. Choose region:")
+	for i:=0; i<len(numbers);i++{
+		fmt.Println("Number ",i," region ",numbers[i])
+	}
+	for !SetRegions(){
+		SetRegions()
+	}
+
+	fmt.Println("Profile: ")
+	for !SetProfile(){
+		SetProfile()
+	}
+
+	}
+
+func SetRegions( ) (ok bool){
+	//get number from the console
+	fmt.Scan(&NumberRegion)
+	if NumberRegion>=0 && NumberRegion<14 {
+	fmt.Println("Your region is: " + numbers[NumberRegion])
+	ok=true
+	return ok
+} else {
+	ok=false
+	fmt.Println("Invalid region, try again")
+	return ok
+}
+}
+
+func SetProfile() (ok bool){
+	//get name of the profile
+	fmt. Scan(&profile)
+	if profile!="" {
+		fmt.Println("Your profile is: " + profile)
+		ok=true
+		return ok
+	} else {
+		ok=false
+		fmt.Println("Invalid profile, try again")
+		return ok
+	}
+}
+
+func MakePath() string{
```

Comment:
It would be better to let user chose where he want to create configuration file. We can print out locations, where Perun looks for configuration files in proper order and then ask user about path.